build.sh

```diff
@@ -623,7 +623,7 @@ function coverage () {
     #Generate individual reports per test
     local DATA_RAW=$(find . -name "*.profraw" -type f)
     local DATA=""
-    local EXCLUDE_REGEX=".*[Tt]est[\/\\]|tiny-process-library.*"
+    local EXCLUDE_REGEX=".*[Tt]est[\/\\]|tiny-process-library.*|ADb[\/\\]include[\/\\]ADb[\/\\]Value\.hpp"
```

Comment:
Perhaps not needed if the fix in CMakeLists.txt would cause the test to be executed.